File: android/app/src/main/kotlin/com/example/spacikopooja/NetworkChangeReceiver.java
package com.example.spacikopooja;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.widget.Toast;

public class NetworkChangeReceiver extends BroadcastReceiver {

    @Override
    public void onReceive(final Context context, final Intent intent) {
        if (checkInternet(context)) {
        }
    }

    boolean checkInternet(Context context) {
        ServiceManager serviceManager = new ServiceManager(context);
        if (serviceManager.isNetworkAvailable()) {
            Toast.makeText(context, "Connected", Toast.LENGTH_SHORT).show();
            return true;
        } else {
            Toast.makeText(context, "Not Connected", Toast.LENGTH_SHORT).show();
            return false;
        }
    }
}

File: android/app/src/main/kotlin/com/example/spacikopooja/NetworkChangeReceiver.kt
package com.example.spacikopooja

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.widget.Toast

class NetworkChangeReceiver : BroadcastReceiver() {
    @Override
    override fun onReceive(context: Context?, intent: Intent?) {
        if (checkInternet(context)) {
        }
    }

    fun checkInternet(context: Context?): Boolean {
        val serviceManager = ServiceManager(context!!)
        return if (serviceManager.isNetworkAvailable) {
            Toast.makeText(context, "Connected", Toast.LENGTH_SHORT).show()
            true
        } else {
            Toast.makeText(context, "Not Connected", Toast.LENGTH_SHORT).show()
            false
        }
    }
}